### PR TITLE
Add Nper() func with respective tests and docs

### PR DESCRIPTION
**Pull request by wowinter13**
Hey, @gyanesh-m !
Would be happy to see your review on this `Nper()` implementation:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -498,3 +500,54 @@ func main() {
...
+```go  
+func Nper(rate float64, pmt float64, pv float64, fv float64, when paymentperiod.Type) float64 
+```  
+Params:  
+```text
+rate  : an interest rate compounded once per period
+pmt   : a (fixed) payment, paid either at the beginning (when =  1)
+        or the end (when = 0) of each period
+pv   : a present value
+fv  : a future value
```
Comment (gyanesh-m) [inline README.md:518]: minor indentation fix needed.

**Review by gyanesh-m — COMMENTED** [review]
Hey @wowinter13 ! Thank you for your contribution to go-financial :) .

**Comment by gyanesh-m** [comment]
@wowinter13 Also, example is missing from [example_reducing_utils_test.go](https://github.com/razorpay/go-financial/blob/master/example_reducing_utils_test.go). Please add the one you added in readme here too so that it gets reflected in go docs.

File: README.md (hunk 1 above)
Comment (wowinter13) [reply README.md]: Fixed

**Comment by wowinter13** [comment]
@gyanesh-m Thanks, fixed:)

File: README.md [hunk 2]
```diff
@@ -498,3 +500,54 @@ func main() {
...
+         or the end (when = 0) of each period
+pv     : a present value
+fv     : a future value
+when   : specification of whether payment is made at the beginning (when = 1)
+         or the end (when = 0) of each period  
+``` 
+
+Nper computes the number of periodic payments.
+
+### Example(Nper)
```
Comment (gyanesh-m) [inline README.md:525]:
```suggestion
### Example(Nper-Loan)
```

File: README.md [hunk 3]
```diff
@@ -498,3 +500,54 @@ func main() {
...
+	pv := float64(5000)
+	when := paymentperiod.ENDING
+
+	nper := gofinancial.Nper(rate, payment, pv, fv, when)
+	fmt.Printf("nper:%v", math.Ceil(nper))
+	// Output:
+	// nper:27
+}
+```
+
```
Comment (gyanesh-m) [inline README.md:553]: add go playground link.

File: README.md (hunk 2 above)
Comment (wowinter13) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (wowinter13) [reply README.md]: Fixed

**Comment by wowinter13** [comment]
@gyanesh-m fixed, thanks once again!

**Review by gyanesh-m — APPROVED** [review]
LGTM.
@wowinter13 Thanks for your contribution to go-financial 🎉 !

**Pull request merged**